1.9/usage/debugging/cli-debugging/debug-subcommands/index.md

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
Merge this entire topic with the current parent topic (`/1.9/usage/debugging/cli-debugging/index.md`).